### PR TITLE
[CIS GCP] Integration test for the project scope

### PR DESCRIPTION
**Pull request by uri-weisman**
### Summary of your changes
Introduce an integration test that will run as part of the pre-merge tasks.

### Related Issues
- Related: https://github.com/elastic/cloudbeat/issues/1322

### Checklist
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary README/documentation (if appropriate)

### CODE REVIEW

File: .github/workflows/cnvm-ci.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ jobs:
       - name: Run Elasticsearch
         uses: elastic/elastic-github-actions/elasticsearch@master
         with:
-          stack-version: 8.8.0
+          stack-version: 8.10.0
```
Comment (oren-zohar) [inline .github/workflows/cnvm-ci.yml:48]: can we get that dynamically instead? @gurevichdmitry

File: .github/workflows/gcp-ci.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      - name: Build cloudbeat binary
+        uses: magefile/mage-action@v3
+        with:
+          version: latest
+          args: build
+
+      - name: Run Elasticsearch
+        uses: elastic/elastic-github-actions/elasticsearch@master
+        with:
+          stack-version: 8.10.0
```
Comment (oren-zohar) [inline .github/workflows/gcp-ci.yml:51]: can we get that dynamically instead? @gurevichdmitry

File: tests/integration/tests/test_sanity_checks.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 import pytest
 from commonlib.utils import get_findings
 
-CONFIG_TIMEOUT = 120
+CONFIG_TIMEOUT = 300
```
Comment (oren-zohar) [inline tests/integration/tests/test_sanity_checks.py:12]: why do you change the timeout?

**Review by oren-zohar — CHANGES_REQUESTED** [review]

File: tests/integration/tests/test_sanity_checks.py (hunk 3 above)
Comment (uri-weisman) [reply tests/integration/tests/test_sanity_checks.py:12]: The workflow failed multiple times to achieve some resources.
Most of the resources are being collected by a single fetcher (gcp_asset_fetcher), therefore, the entire flow takes more time than AWS CSPM.

File: .github/workflows/gcp-ci.yml (hunk 2 above)
Comment (gurevichdmitry) [reply .github/workflows/gcp-ci.yml]: @oren-zohar We can use the environment variable `ELK_VERSION`.

File: tests/integration/tests/test_sanity_checks.py (hunk 3 above)
Comment (gurevichdmitry) [reply tests/integration/tests/test_sanity_checks.py:12]: We can manage a separate timeout definition for GCP findings using `GCP_CONFIG_TIMEOUT`.

**Review by gurevichdmitry — APPROVED** [review]
@uri-weisman LGTM. Do you have an example of a successful GCP workflow run after all the changes?

**Comment by uri-weisman** [comment]
> @uri-weisman LGTM. Do you have an example of a successful GCP workflow run after all the changes?

https://github.com/uri-weisman/cloudbeat/actions/runs/6337782149

**Review by oren-zohar — APPROVED** [review]
LGTM

**Pull request merged**